pybamm/models/submodels/interface/open_circuit_potential/plett_ocp.py

```diff
@@ -0,0 +1,151 @@
+#
+# from Wycisk 2022
+#
+import pybamm
+from . import BaseOpenCircuitPotential
+
+
+class PlettOpenCircuitPotential(BaseOpenCircuitPotential):
+    def get_fundamental_variables(self):
+        domain, Domain = self.domain_Domain
+        phase_name = self.phase_name
+        h = pybamm.Variable(
+            f"{Domain} electrode {phase_name}hysteresis state",
+            domains={
+                "primary": f"{domain} electrode",
+                "secondary": "current collector",
+            },
+        )
+        return {
+            f"{Domain} electrode {phase_name}hysteresis state": h,
+        }
+
+    def get_coupled_variables(self, variables):
+        domain, Domain = self.domain_Domain
+        phase_name = self.phase_name
+        phase = self.phase
+
+        # get dQ, change in capacity with respect to stoichiometry
+        c_max = self.phase_param.c_max
+        epsilon_s_av = self.phase_param.epsilon_s_av
+        V_electrode = self.param.A_cc * self.domain_param.L
+        Li_max = c_max * V_electrode * epsilon_s_av
+        Q_max = Li_max * self.param.F / 3600
```

Comment:
There is already a variable `f"{Domain} electrode capacity [A.h]"` defined in the active material model that does this calculation also accounting for any change in active material fraction

Reply:
Originally I was calculating dQ(sto) by means of a function in the lithium ion parameters (main parameters for a phase I believe), but @valentinsulzer suggested placing it in the submodel class as a coupled variable. 

Since dQ(sto) should be a constant value, I realized this could just be a scalar. Although, I am not sure whether it needs to be an equation as d(sto) may be different depending on the number of steps during solving. I still haven't tested this (looked at actual data or results) with the change from I made, so I am not sure if I will actually get the results I expect. All the more reason to make an example notebook.

I will make the example notebook and assess whether the function form of Q was necessary or not.